Index: calc-price/src/main/java/Brest/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- calc-price/src/main/java/Brest/Main.java	(revision 8c1e11dbfa231c0765525c85fafcd59bf6b772df)
+++ calc-price/src/main/java/Brest/Main.java	(date 1580923115111)
@@ -12,7 +12,7 @@
      int i = 0;
      do{
          if(i == 0){
-             System.out.println("Please, enter distance or q for exit: ");
+             System.out.println(" enter distance or q for exit: ");
          } else if(i == 1){
              System.out.println("Please, enter price per km or q for exit: ");
          } else if(i == 2){
